server/api/index.js

```diff
@@ -1,12 +1,12 @@
 const { Router } = require('express');
 
 const items = [
-  { id: 1, name: 'item 1' },
-  { id: 2, name: 'item 2' },
-  { id: 3, name: 'item 3' },
-  { id: 4, name: 'item 4' },
-  { id: 5, name: 'item 5' },
-  { id: 6, name: 'item 6' },
+  { id: 1, name: ['Item1', '/images/placeholder.png', 100, undefined] },
```

Comment:
Could we add more fields to the object instead?

```diff
- { id: 1, name: ['Item1', '/images/placeholder.png', 100, undefined] },
+ { id: 1, name: 'Item1', imageUrl: '/images/placeholder.png', originalPrice: 100, discountedPrice: undefined },
```

Reply:
Okay!